pom.xml

```diff
@@ -1399,5 +1407,19 @@
         </pluginManagement>
       </build>
     </profile>
+    <profile>
```

Comment:
I think it is important to have these. Is there really a burden in having these in the pom?